thousandeyes/schemas.go

```diff
@@ -1136,6 +1134,20 @@ var schemas = map[string]*schema.Schema{
 			},
 		},
 	},
+	"tests-label": {
```

Comment:
There's a schema for `tests` right above this one, but I'm not sure what it is being used for, as I don't think it was being used for Labels. I decided to add a new one for the Labels use case, similar to the `agents-label`. 

At some point we might want to reconsider this "shared" schemas file as we're starting to create many exceptions.